src/azure-cli/azure/cli/command_modules/network/_util.py

```diff
@@ -66,3 +67,31 @@ def delete_func(cmd, resource_group_name, resource_name, item_name, no_wait=Fals
     func_name = 'delete_network_resource_property_entry_{}_{}'.format(resource, prop)
     setattr(sys.modules[__name__], func_name, delete_func)
     return func_name
+
+
+# workaround for : https://github.com/Azure/azure-cli/issues/17071
+def delete_lb_resource_property_entry(resource, prop):
+    """ Factory method for creating delete functions. """
+
+    def delete_func(cmd, resource_group_name, resource_name, item_name, no_wait=False):  # pylint: disable=unused-argument
+        client = getattr(network_client_factory(cmd.cli_ctx), resource)
+        item = lb_get(client, resource_group_name, resource_name)
+
+        if item.__getattribute__(prop) is not None:
+            keep_items = [x for x in item.__getattribute__(prop) if x.name.lower() != item_name.lower()]
+        else:
+            keep_items = None
+
+        with cmd.update_context(item) as c:
+            c.set_param(prop, keep_items)
+        if no_wait:
+            sdk_no_wait(no_wait, client.begin_create_or_update, resource_group_name, resource_name, item)
+        else:
+            result = sdk_no_wait(no_wait, client.begin_create_or_update,
+                                 resource_group_name, resource_name, item).result()
+            if next((x for x in getattr(result, prop) or [] if x.name.lower() == item_name.lower()), None):
+                raise CLIError("Failed to delete '{}' on '{}'".format(item_name, resource_name))
+
+    func_name = 'delete_network_resource_property_entry_{}_{}'.format(resource, prop)
```

Comment:
Should be `delete_lb_resource_property_entry_`

Reply:
good job. Will be fixed in another PR